### PR TITLE
Misc release

### PR DESCRIPTION
**Pull request by mfkl**

### CODE REVIEW

**Comment by mfkl** [comment]
Some tests are failing on Windows because of `The SDK 'Microsoft.NET.SDK.WorkloadAutoImportPropsLocator' specified could not be found.` Investigating.

File: src/SolutionValidator/ProjectLoader.cs [hunk 1]
```diff
@@ -50,12 +50,7 @@ public ProjectLoader(ILogger<ProjectLoader> logger)
 
             if (sdkVersion >= new Version(6, 0))
             {
-#if NET6_0_OR_GREATER
-                Environment.SetEnvironmentVariable("MSBUILDADDITIONALSDKRESOLVERSFOLDER", Path.Combine(sdkDirectory, "SdkResolvers"));
-                Environment.SetEnvironmentVariable("MSBuildEnableWorkloadResolver", "true");
-#else
                 Environment.SetEnvironmentVariable("MSBuildEnableWorkloadResolver", "false");
-#endif
```
Comment (mfkl) [inline src/SolutionValidator/ProjectLoader.cs:58]: Any thoughts on this @wo-roberts? It fixes the tests on Windows but I'm not quite sure this is what is expected.

**Comment by marcin-krystianc** [comment]
I think that using the `MSBuildLocator` is a solution to all MSBuild/SDK related issues, see https://github.com/G-Research/Tack/pull/2/commits/65b7942f2d0671123d6ab7353730b746fa658b24

**Comment by mfkl** [comment]
> I think that using the `MSBuildLocator` is a solution to all MSBuild/SDK related issues, see [G-Research/Tack@65b7942](https://github.com/G-Research/Tack/commit/65b7942f2d0671123d6ab7353730b746fa658b24)

Thanks, used your code from that commit to do the same here.

**Comment by mfkl** [comment]
Initial CI run is ✔️  https://github.com/mfkl/SolutionValidator/actions/runs/5554815419

File: tests/SolutionValidator.Tests/SolutionValidator.Tests.csproj [hunk 2]
```diff
@@ -19,6 +19,7 @@
     <PackageReference Include="FakeItEasy" Version="6.0.1" />
     <PackageReference Include="Divergic.Logging.Xunit" Version="3.5.1" />
     <PackageReference Include="coverlet.collector" Version="1.3.0" />
+	<PackageReference Include="Microsoft.Build.Locator" Version="1.5.5"/>
```
Comment (marcin-krystianc) [inline tests/SolutionValidator.Tests/SolutionValidator.Tests.csproj:22]: nit:formatting issue

**Review by marcin-krystianc — APPROVED** [review]

File: tests/SolutionValidator.Tests/SolutionValidator.Tests.csproj (hunk 2 above)
Comment (mfkl) [reply tests/SolutionValidator.Tests/SolutionValidator.Tests.csproj]: done

**Review by marcin-krystianc — APPROVED** [review]

**Pull request merged**